pkg/snowflake/stage_test.go

```diff
@@ -68,6 +71,12 @@ func TestStageChangeCredentials(t *testing.T) {
 	a.Equal(s.ChangeCredentials("aws_role='arn:aws:iam::001234567890:role/mysnowflakerole'"), `ALTER STAGE "test_db"."test_schema"."test_stage" SET CREDENTIALS = (aws_role='arn:aws:iam::001234567890:role/mysnowflakerole')`)
 }
 
+func TestStageChangeStorageIntegration(t *testing.T) {
+	a := assert.New(t)
+	s := Stage("test_stage", "test_db", "test_schema")
+	a.Equal(s.ChangeStorageIntegration("MY_INTEGRATION"), `ALTER STAGE "test_db"."test_schema"."test_stage" SET STORAGE_INTEGRATION = 'MY_INTEGRATION'`)
```

Comment:
I was looking to build Storage integration and found this. I went through the test and found that the test assertion was _mildly_ wrong. Before that, the usage of `assert.Equal` is reversed in terms of expected & actual, hence the error message from assertion is flipped. 

Actual error from Travis
```--- FAIL: TestStageChangeStorageIntegration (0.00s)
    stage_test.go:77: 
        	Error Trace:	stage_test.go:77
        	Error:      	Not equal: 
        	            	expected: "ALTER STAGE \"test_db\".\"test_schema\".\"test_stage\" SET STORAGE_INTEGRATION = MY_INTEGRATION"
        	            	actual  : "ALTER STAGE \"test_db\".\"test_schema\".\"test_stage\" SET STORAGE_INTEGRATION = 'MY_INTEGRATION'"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-ALTER STAGE "test_db"."test_schema"."test_stage" SET STORAGE_INTEGRATION = MY_INTEGRATION
        	            	+ALTER STAGE "test_db"."test_schema"."test_stage" SET STORAGE_INTEGRATION = 'MY_INTEGRATION'
        	Test:       	TestStageChangeStorageIntegration```

Looks like `MY_INTEGRATION` in the "expected" string "ALTER STAGE ..." should NOT be Single quoted :)

@sjauld - Please see if you can do this minor edit. And, thank you for spending time on this.

Reply:
Thanks @brewkode! This was a bad oversight on my behalf.

Note that all of the assertions are the wrong way around so I'll just leave these for now.